src/render/mod.rs

```diff
@@ -724,7 +723,8 @@ pub fn render_app_builder(app: &mut App) {
         )
         .add_systems(
             Render,
-            (queue_infinite_grids, queue_grid_view_bind_groups).in_set(RenderSet::Queue),
+            (queue_infinite_grids, queue_grid_view_bind_groups)
+                .in_set(RenderSet::PrepareBindGroups),
```

Comment:
I'm not sure I understand why this is necessary. I understand that the sets were reordered but the Queue set should still be used to queue things. I haven't touched this code that much though so it's possible it's doing something incompatible with that.

Reply:
The problem was that you put bind group creation and queuing into the same system, while Bevy 0.12 wants them to be in separate systems. I've split up the logic appropriately.

Reply:
Ah, thank you for the investigation, I didn't implement the crate originally so I wasn't aware of that.